### ISSUE TITLE
BattlegroupSelector.csx requires debug mode but does not enable it

### ISSUE THREAD

**Issue by OceanBagel** [krzys-h/UndertaleModTool#701]
The script BattlegroupSelector.csx requires debug to be enabled to function, but the script does not enable debug mode. There are two potential solutions. One is to modify the script to not require debug mode, and the other is to modify the script to enable debug mode.

**Comment by Archie-osu** [comment]
This is true for other scripts too, so if one of these suggested changes go through, it should be applied to all other scripts too.
